server/recceiver/cfstore.py

```diff
@@ -52,35 +55,47 @@ def startService(self):
             Using the default python cf-client.  The url, username, and
             password are provided by the channelfinder._conf module.
             """
+            # TODO: Explain why are we importing CF client here.
```

Comment:
This module is always imported by `twistd`, even when 'proc = cf' isn't given.  This way the ChannelFinderClient import is only attempted if it is actually needed.